### ISSUE TITLE
Can we check the Livox Internal IMU working good or not?

### ISSUE THREAD

**Issue by rahulsharma11** [gaoxiang12/faster-lio#19]
Hi,
Hope you are doing well.

I used Livox Avia with Fast-LIO and faster-LIO algorithms (no External IMU) with my laptop. 

Issue-
Picture below this is a top view of 3 sides scanned pillar with your as well as Fast_LIO algorithm just after start.
![WhatsApp Image 2022-05-18 at 5 35 04 PM](https://user-images.githubusercontent.com/28132746/169034885-bee67ab0-a5b2-4f43-a292-408fd50bf915.jpeg)

This is correctly mapped as all 3 edges are alighned.

But after sometime (around 30 mins of run), my pillar starts appearing like this-
![WhatsApp Image 2022-05-14 at 10 25 06 PM](https://user-images.githubusercontent.com/28132746/169035030-99b44b2b-cf54-4c2f-9bbb-3c42a4f0613a.jpeg)

Here you can see the double wall effect. Its like after few mins, the IMU alighnment is getting OFF hence mapping is getting OFF.

I suspect internal IMU got off.
Is there any utility so that i can check whether the interanal IMU is working within a perfect range? Do i need to recalibrate the IMU?
I checked the Livox-viewer utility. Raw data from Livox is fine.
Basically i want to get proof if IMU is the main culpit or something else.

Thanks.

**Comment by rahulsharma11** [comment]
Hi @gaoxiang12 ,
Thanks for the pointers.

I would like to add that this issue is unexpected for time. Like sometime it starts happening even after 10 mins. So i only suspects that Lidar IMU is creating problem or not. One more thing FastLIO was already working from last few months, this behaviour is all of sudden appeared and it is now there for any scan. Like Hardware issue.

**Comment by rahulsharma11** [comment]
Also there is a doubt for "imu_rate". It is recommended that imu_rate should be 1. if it is 0 then it will not push the imu data.

But i have been running the default livox_lidar_config.json where imu_rate=0. But still i am getting the /livox/imu data.
So will this effect on imu quality ?
